doc/content/ttn-lw-cli/ttn-lw-cli_use.md

```diff
@@ -14,12 +14,14 @@ ttn-lw-cli use [flags]
 ### Options
 
 ```
-      --fetch-ca        Connect to server and retrieve CA
-      --grpc-port int   
-  -h, --help            help for use
-      --host string     Server host name (default "localhost")
-      --overwrite       Overwrite existing config files
-      --user            Write config file in user config directory
+      --fetch-ca                      Connect to server and retrieve CA
```

Comment:
The diff here looks a bit weird. `oauth-server-address` and `insecure` are global and should not have changed.

Reply:
I checked out `v3.25.0` branch in `lorawan-stack` clone while doing this, is that ok?

Reply:
I was wrong - you did everything right. It seems that when we updated the dependencies we've met a change in the way `cobra` handles the local vs global parameters.

References https://github.com/TheThingsNetwork/lorawan-stack/pull/2420